packages/react-sdk/src/components/Tooltip/WithTooltip.tsx

```diff
@@ -1,25 +1,29 @@
-import { ComponentProps, useState } from 'react';
+import { ComponentProps, createContext, useContext, useState } from 'react';
 import { Tooltip, TooltipProps } from './Tooltip';
 import { useEnterLeaveHandlers } from './hooks';
 
 type WithPopupProps = ComponentProps<'div'> &
   Omit<TooltipProps<HTMLDivElement>, 'referenceElement'>;
 
+export const TooltipContext = createContext<{
+  hideTooltip?: () => void;
+}>({});
+
 // todo: duplicate of CallParticipantList.tsx#MediaIndicator - refactor to a single component
 export const WithTooltip = ({
   title,
   tooltipClassName,
   tooltipPlacement,
   ...props
 }: WithPopupProps) => {
-  const { handleMouseEnter, handleMouseLeave, tooltipVisible } =
+  const { handleMouseEnter, handleMouseLeave, tooltipVisible, forceHide } =
     useEnterLeaveHandlers<HTMLDivElement>();
   const [tooltipAnchor, setTooltipAnchor] = useState<HTMLDivElement | null>(
     null,
   );
 
   return (
-    <>
+    <TooltipContext.Provider value={{ hideTooltip: forceHide }}>
```

Comment:
Sometimes we want to hide tooltip even if the target is still hovered. One example is if it conflicts with an open menu.

https://github.com/GetStream/stream-video-js/assets/975978/b095b746-15ef-4932-a375-80e958997fef